modules/sdk-unified-wallet/README.md

```diff
@@ -1 +1,36 @@
-# Bigto sdk-evm
+# BitGo sdk-unified-wallet
+
+Example: Generate an EVM wallet
+
+```
+const BitGoJS = require('bitgo');
+const { EcdsaEVMUnifiedWallets } = require('@bitgo/sdk-unified-wallet');
+
+const bitgo = new BitGoJS.BitGo({
```

Comment:
maybe show an example using the newer format with the BitGoAPI class?

Reply:
Updated